test/browser/run-test.sh

```diff
@@ -44,9 +44,31 @@ if [ "$ID" = "rhel" ]; then
     "
 fi
 
-if [ "$TEST_OS" = "centos-8-stream" ]; then
-    EXCLUDES="$EXCLUDES TestMachinesConsoles.testExternalConsole"
-fi
+# We only have one VM and tests should take at most one hour. So run those tests which exercise external API
+# (and thus are useful for reverse dependency testing and gating), and exclude those which test internal
+# functionality -- upstream CI covers that.
+EXCLUDES="$EXCLUDES
+          TestMachinesCreate.testDisabledCreate
+
+          TestMachinesConsoles.testExternalConsole
+          TestMachinesConsoles.testInlineConsole
+          TestMachinesConsoles.testInlineConsoleWithUrlRoot
+          TestMachinesConsoles.testSerialConsole
+          TestMachinesConsoles.testSwitchConsoleFromSerialToGraphical
+
+          TestMachinesDisks.testAddDiskAdditionalOptions
+          TestMachinesDisks.testAddDiskCustomPath
+          TestMachinesDisks.testDetachDisk
+          TestMachinesDisks.testDiskEdit
+
+          TestMachinesLifecycle.testBasicLibvirtUserUnprivileged
+          TestMachinesLifecycle.testBasicWheelUserUnprivileged
+          TestMachinesLifecycle.testDelete
+          TestMachinesLifecycle.testLibvirt
+
+          TestMachinesNetworks.testNetworkSettings
+          TestMachinesNetworks.testNetworkState
+          "
 
```

Comment:
I would recommend adding also
`TestMachinesHostDevs.testHostDevAddMultipleDevices` as it is using the same API as `TestMachinesHostDevs.testHostDevAddSingleDevice`. `testHostDevAddSessionConnection` again using the same API, only with different connection parameter: system/session

Reply:
Thanks, I'll exclude `testHostDevAddMultipleDevices`. The session one feels more prone to regressions, as there's a lot of device/service permissions in between. We can cut that once we get over the 50 minute line, but until then I'd rather leave it running.